src/main/java/com/crafting/Knowledge.java

```diff
@@ -0,0 +1,45 @@
+package com.crafting;
+
+import com.minecolonies.util.InventoryUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.util.InventoryFunctions'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)